src/sparseml/pytorch/sparsification/quantization/quantize_qat_export.py

```diff
@@ -1569,9 +1589,8 @@ def quantize_torch_qat_export(
     _convert_quantizable_gemm_no_activations(model)
     _quantize_qat_embedding(model)
     quantize_resnet_identity_add_inputs(model)
-    quantized_residual_add_optim(model)
     _remove_duplicate_quantize_ops(model)
-    _cleanup_unused_quants(model)
+    #_cleanup_unused_quants(model)
```

Comment:
Is this comment intentional? Could we remove this before merging?

Reply:
Removed